pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -44,6 +44,13 @@ import (
 )
 
 const (
+	// maxRetries is the number of times a service will be retried before it is dropped out of the queue.
+	// With the current rate-limiter in use (5ms*2^(maxRetries-1)) the following numbers represent the times
```

Comment:
- [ ] s/the times a deployment is going to be requeued/the sequence of delays between each successive queuing of a deployment/